### PR TITLE
Centcom starts importing cat ears/tails

### PR DESCRIPTION
**Pull request by improvedname**
we all know centcom is half catmen and catgirls this makes sense lore wise

:cl: Improvedname
add: You can now export cat ears/tails for 1000 credits
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:) LORE

### CODE REVIEW

**Comment by kevinz000** [comment]
WHY ARE YOU BUFFING CATS?!!

**Comment by RemieRichards** [comment]
They need to import lizard parts too then.

@kevinz000 it's a nerf, because people will remove their tails for the credits.

**Comment by WJohn** [comment]
remove catmemes when

**Comment by improvedname** [comment]
someday

**Comment by Grazyn** [comment]
All-cat cargo crew new meta

**Comment by meeeeme** [comment]
Crate of cat parts in surgery when?

**Review by Cyberboss — APPROVED** [review]

File: code/modules/cargo/exports/organs.dm [hunk 1]
```diff
@@ -75,7 +75,22 @@
 	if(C.preserved)
 		return ..() * 2
 	return ..()
+	
+// Kitty Organs.
 
+/datum/export/organ/cat
+	contrabrand = FALSE
+	include_subtypes = FALSE
```
Comment (Cyberboss) [inline code/modules/cargo/exports/organs.dm]: This datum isn't necessary

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by lzimann** [comment]
kill cat get cash

**Comment by Niknakflak** [comment]
we can remove cat memes when we remove xeno memes first

**Comment by ExcessiveUseOfCobblestone** [comment]
> we can remove cat memes when we remove xeno memes first

I'm down.

**Comment by Iamgoofball** [comment]
@Niknakflak Unfortunately both fetishes have maintainers who close PRs removing them.

**Comment by Tacolizard** [comment]
@RemieRichards i'm pretty sure you can export lizard parts. The skin at least.

**Comment by RemieRichards** [comment]
@Tacolizard You have to kill for skin, you don't for bodyparts, if people want to make species more consistent, then it's only fair to be able to sell lizard tails and such also.

**Comment by Iamgoofball** [comment]
censorship

@Niknakflak Both """features""" have maintainers actively protecting them, stating this as a fact is apparently discouraged but whatever.

**Comment by Iamgoofball** [comment]
also

:+1:

this means catgrills have actual gameplay impact and I can use that as an excuse to remove them

ps. screenshotting all comments from here on for when the FAKE NEWS MSM censors me

**Comment by Tacolizard** [comment]
add a crate of cat ears and tails to the brig execution chamber/torture chamber when

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**